docs/OverseeingIssuePrLifeCycle.md

```diff
@@ -16,22 +16,19 @@ When an issue comes in,
 
 # Assigning reviewers for PRs
```

Comment:
Shall we avoid the use of `assign` for this section ([line 23](https://github.com/oss-generic/process/pull/62/files#diff-8663d1e7fd254bc258c2c7c8b8d8f337R23)) and rename to `Requesting reviews for PRs`?